### PR TITLE
[13.0] [MIG] partner_second_lastname

### PR DESCRIPTION
**Pull request by emagdalenaC2i**

### CODE REVIEW

File: partner_second_lastname/tests/__init__.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 from . import test_name
-from . import test_onchange
```
Comment (rousseldenis) [inline partner_second_lastname/tests/__init__.py]: Why removing these tests?

Comment (emagdalenaC2i) [reply partner_second_lastname/tests/__init__.py]: There were an issue with it and I think it is not needed
It was also removed in #805

**Review by HaraldPanten — CHANGES_REQUESTED** [review]
Something seems not being working properly.

1- Try to create a contact. You'll see:

`Odoo Server Error
Traceback (most recent call last):
  File "/.repo_requirements/odoo/odoo/http.py", line 624, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/.repo_requirements/odoo/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/.repo_requirements/odoo/odoo/tools/pycompat.py", line 14, in reraise
    raise value
  File "/.repo_requirements/odoo/odoo/http.py", line 669, in dispatch
    result = self._call_function(**self.params)
  File "/.repo_requirements/odoo/odoo/http.py", line 350, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 915, in __call__
    return self.method(*args, **kw)
  File "/.repo_requirements/odoo/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/.repo_requirements/odoo/addons/web/controllers/main.py", line 1322, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/.repo_requirements/odoo/addons/web/controllers/main.py", line 1314, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 387, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/.repo_requirements/odoo/odoo/api.py", line 374, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/.repo_requirements/odoo/odoo/models.py", line 6095, in onchange
    record._onchange_eval(name, field_onchange[name], result)
  File "/.repo_requirements/odoo/odoo/models.py", line 5896, in _onchange_eval
    method_res = method(self)
  File "/home/odoo/build/OCA/partner-contact/partner_second_lastname/models/res_partner.py", line 112, in _onchange_subnames
    super(ResPartner, self)._onchange_subnames()
AttributeError: 'super' object has no attribute '_onchange_subnames'`

**Comment by emagdalenaC2i** [comment]
@HaraldPanten Done

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by ValentinVinagre — APPROVED** [review]
LGTM 👍

**Comment by ValentinVinagre** [comment]
Hi @rousseldenis ,
it's ready to merge?

**Comment by rousseldenis** [comment]
@emagdalenaC2i Can you squash your commits ?

**Comment by emagdalenaC2i** [comment]
@rousseldenis done

**Comment by emagdalenaC2i** [comment]
@rousseldenis could you check again?

**Comment by NL66278** [comment]
Closing and opening again to recheck

**Pull request merged**